src/components/Navbar/Navbar.test.js

```diff
@@ -0,0 +1,273 @@
+                </ConfigContext.Provider>
+            </StateContext.Provider>
+        );
+        expect(getByTestId('ButtonFavorites')).toHaveTextContent('Favorites');
+    });
+
+    it('Should have text "Oracle" ', () => {
+        const { getByTestId } = render(
+            <StateContext.Provider value={providerPropsS} >
+                <ConfigContext.Provider value={providerPropsC} >
+                    <VideoContext.Provider value={providerPropsV}>
+                        <Navbar/>
+                    </VideoContext.Provider>
+                </ConfigContext.Provider>
+            </StateContext.Provider>
+        );
+        expect(getByTestId('InputNavBar')).toHaveValue('Oracle');
+    });
+
+    it('Dark mode should have this style ', () => {
+        const { getByTestId } = render(
+            <StateContext.Provider value={providerPropsS} >
+                <ConfigContext.Provider value={providerPropsC} >
+                    <VideoContext.Provider value={providerPropsV}>
+                        <Navbar/>
+                    </VideoContext.Provider>
+                </ConfigContext.Provider>
+            </StateContext.Provider>
+        );
+        expect(getByTestId('DarkMode')).toHaveStyle({ color: "white" });
+    });
+
+    it('Switch should be "true" ', () => {
+        const { getByTestId } = render(
+            <StateContext.Provider value={providerPropsS} >
+                <ConfigContext.Provider value={providerPropsC} >
+                    <VideoContext.Provider value={providerPropsV}>
+                        <Navbar/>
+                    </VideoContext.Provider>
+                </ConfigContext.Provider>
+            </StateContext.Provider>
+        );
+        expect(getByTestId('SwitchNavBar')).toBeChecked();
+    });
+    
+    it('Login icon should be there', () => {
+        const { getByTestId } = render(
+            <StateContext.Provider value={providerPropsS} >
+                <ConfigContext.Provider value={providerPropsC} >
+                    <VideoContext.Provider value={providerPropsV}>
+                        <Navbar/>
+                    </VideoContext.Provider>
+                </ConfigContext.Provider>
+            </StateContext.Provider>
+        );
+        expect(getByTestId('LogoutIcon')).toBeInTheDocument();
+    });
+
+});
+
+describe('Test some actions', () => {
+
+    // beforeAll(() => {
+    //     Object.defineProperty(window, 'matchMedia', {
+    //         writable: true,
+    //         value: jest.fn().mockImplementation(query => ({
+    //           matches: false,
+    //           media: query,
+    //           onchange: null,
+    //           addListener: jest.fn(), // Deprecated
+    //           removeListener: jest.fn(), // Deprecated
+    //           addEventListener: jest.fn(),
+    //           removeEventListener: jest.fn(),
+    //           dispatchEvent: jest.fn(),
+    //         })),
+    //     });
+    // })
```

Comment:
Maybe you forgot to remove the commented code, but, in any case, I recommend to delete it.